microsoft-edge/devtools-guide-chromium/console/console-debug-javascript.md

```diff
@@ -23,7 +23,9 @@ Open DevTools in the browser.  The **Open Console to view errors** button on the
    DevTools gives detailed information about the error in the **Console**  
 :::image-end:::  
 
-The information suggests that the error is on line 16 of the `error.html` file.  Select the `error.html:16` link on the right of the **Console**.  The **Sources** tool opens and highlights the line of code with the error.  
+Many error messages in the **Console** have a **Search for this message on the Web** button, shown as a magnifying glass.  This feature was introduced in Microsoft Edge version 94.  For more information, navigate to [Look up error messages on the web from the Console][LookupErrorsWebFromConsole].
```

Comment:
Same comment here as in the `console\index` file. We should just align on one of the options between:
- `error strings`
- `error messages`
- `errors`
and use that consistently between What's New and this doc

Reply:
Let's stick with error messages.

Reply:
In devtools.md, changed "error string" or "error message" to "error".
In console/index.md and console/debug-javascript.md, changed "error string" to "error message".